File: Web/src/main/java/com/itechart/webflux/web/core/model/entity/Message.java
package com.itechart.webflux.web.core.model.entity;

import lombok.Data;
import org.springframework.data.annotation.Id;

import java.util.Date;

@Data
public class Message implements Entity {

    @Id
    private String id;
    private String sender;
    private String receiver;
    private String text;
    private Date date;
    private boolean exists;

}

File: Web/src/main/java/com/itechart/webflux/web/core/model/entity/Message.kt
package com.itechart.webflux.web.core.model.entity

import org.springframework.data.annotation.Id
import org.springframework.data.mongodb.core.mapping.Document
import java.util.*

@Document(collection = "message")
class Message(@Id override var id: String?, var sender: String?, var receiver: String?, var text: String?, var date: Date, var exists: Boolean) : Entity
